controller/appcontroller.go

```diff
@@ -1037,6 +1025,10 @@ func (ctrl *ApplicationController) processRequestedAppOperation(app *appv1.Appli
 		} else {
 			logCtx.Warnf("Fails to requeue application: %v", err)
 		}
+		// write back to informer in order to avoid stale cache
```

Comment:
Please remove this change. Method `setOperationState` is already updating the informer.